docs/configuration.md

```diff
@@ -99,30 +99,43 @@ the WebSocket endpoint is not exposed.
 
 ### Network ID
 
-The release package is preconfigured with mainnet network_id. Please change the configuration to interact with testnet.
-The testnet (internally called UAT) has the network ID `ae_uat` - this is set in the configuration:
+The network that node connects to can be change by setting `fork_management` > `network_id` in the configuration file.
+The default network that node package is preconfigured with is **mainnet** with ID `ae_mainnet`.
+
+The **testnet** (internally called UAT) has the network ID `ae_uat`. To join the testnet set `network_id` to `ae_uat` in the configuration:
 
 ```yaml
 fork_management:
     network_id: ae_uat
 ```
-For mainnet network the network ID defaults to `ae_mainnet`.
 
 ### Block reward initiative
 
-The new schema of managing block reward can be tuned with following configuration parameters.
-Mainnet values are included in the base code, however for testnet use the following values - to be added in the `chain` section of the config:
+The BRI beneficiaries configuration can be changed with following configuration parameters in the `chain` section of the config:
 
 ```yaml
     protocol_beneficiaries_enabled: true
     protocol_beneficiaries: ["ak_2A3PZPfMC2X7ZVy4qGXz2xh2Lbh79Q4UvZ5fdH7QVFocEgcKzU:109"]
 ```
 
+This example refers to mainnet configuration.
+Note that this is under consensus and changing it to anything else will prevent your node to participate in mainnet.
+
+Mainnet and testnet values are automatically set based on [Network ID](#network-id) configuration above, so it don't need to be configured explicitly.
```

Comment:
So maybe we shall delete the whole `### Block reward initiative` subsection?

Reply:
I agree. Let's remove it if other reviewers agree on this @sennui ?